daml-lf/archive/encoder/src/main/scala/com/daml/lf/archive/DarWriter.scala

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.lf.archive
+
+import java.io.{ByteArrayOutputStream, OutputStream}
+import java.util.zip.{ZipEntry, ZipOutputStream}
+import scalaz.syntax.traverse._
```

Comment:
just wondering where are you using that.

Reply:
`Dar.map`

Reply:
:+1: